Source/displayinfo/include/ExtendedDisplayIdentification.h

```diff
@@ -199,8 +581,8 @@ namespace Plugin {
         // -------------------------------------------------------------
         // Accessors to the base information of the EDID raw buffer.
         // -------------------------------------------------------------
-        string Manufacturer() const {
-            string result;
+        std::string Manufacturer() const {
+            std::string result;
```

Comment:
Read the discussion above but why would this functionality not still include the module file. If you start using this library, you are dependend on Thunder (at least the com library and thus all lower layers) for communication and we could still use the Wide character features offered from the Thunder portability layer. Now you break this architectural feature. So please use the already used Thunder dependency and use the string, defined in Thunder.

Reply:
It appears to be only a utility and does not dependent on any Thunder functionality (maybe except ASSERT). It's something used to 'parse' data we get from DisplayInfo client lib

Reply:
I reverted back to TCHAR and string with including the Module.h